src/pages/index.js

```diff
@@ -31,20 +24,52 @@ const cardSubmitBtn = cardModal.querySelector(".modal__submit-btn");
 const cardNameInput = cardModal.querySelector("#add-card-name-input");
 const cardLinkInput = cardModal.querySelector("#add-card-link-input");
 
+const avatarModal = document.querySelector("#avatar-modal");
+const avatarModalCloseBtn = avatarModal.querySelector("#modal-close-btn");
+const avatarForm = avatarModal.querySelector(".modal__form");
+const avatarSubmitBtn = avatarModal.querySelector(".modal__submit-btn");
+const avatarInput = avatarModal.querySelector("#profile-avatar-input");
+
 const previewModal = document.querySelector("#preview-modal");
 const previewModalImageEl = previewModal.querySelector(".modal__image");
 const previewModalCaptionEl = previewModal.querySelector(".modal__caption");
 const modalCloseTypePreview = previewModal.querySelector(".modal__close-btn_type_preview");
 
-// Initial card rendering
-initialCards.forEach((item) => {
-  const cardElement = getCardElement(item);
-  cardsList.append(cardElement);
-});
+
+const deleteModal = document.querySelector("#delete-modal");
+const deleteModalXBtn = deleteModal.querySelector(".modal__close-btn_type_preview");
+
+const deleteBtn = deleteModal.querySelector(".modal__submit-btn");
+const deleteModalCloseBtn = deleteModal.querySelector("#modal-delete-close-btn");
+
+let selectedCard;
+let selectedCardId;
+
 enableValidation(settings);
 
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "92dcf13b-9213-4fca-8c66-c8527311427d",
+    "Content-Type": "application/json",
+  },
+});
+
+api.getAppInfo()
+  .then(([cards]) => {
+    cards.forEach((item) => {
+      const cardElement = getCardElement(item);
+      cardsList.append(cardElement);
+    });
+  })
+  .catch(console.error);
+
+api.getUserInfo().then((user) => {
```

Comment:
`api.getUserInfo()` is not needed in line 67 because you get `user` in line 59 inside the `api.getAppInfo()` call